src/client.js

```diff
@@ -117,7 +116,7 @@ export async function connect(myVatID, addr, commandfile) {
   pullStream(
     conn,
     pullSplit('\n'),
-    pullStream.map(line => {
```

Comment:
I don't think `pullStream` offers a `forEach` method (`pullStream` is a funky stream thing, not an `Array`, and `pullStream.map` is not just `Array.map`). I don't think we have any tests covering this tool, but I suspect this would break. We could have it return a dummy value if eslint thinks this is an `Array.map` and we need to appease it.

Reply:
Ah, good point, I'll change that